tests/metrics/test_scoring.py

```diff
@@ -166,3 +170,52 @@ def test_raises(self, metric: MetricAtK, arg_names: tp.List[str]) -> None:
         kwargs = {name: getattr(self, name) for name in arg_names}
         with pytest.raises(ValueError):
             calc_metrics({"m": metric}, **kwargs)
+
+    def test_success_debias(self) -> None:
+        debias_config = DebiasConfig(iqr_coef=1.5, random_state=32)
+        debias_metrics = {
+            "debias_precision@3": Precision(k=3, debias_config=debias_config),
+            "debias_rprecision@3": Precision(k=3, r_precision=True, debias_config=debias_config),
+            "debias_recall@3": Recall(k=3, debias_config=debias_config),
+            "debias_f1beta@3": F1Beta(k=3, debias_config=debias_config),
+            "debias_accuracy@3": Accuracy(k=3, debias_config=debias_config),
+            "debias_mcc@3": MCC(k=3, debias_config=debias_config),
+            "debias_hitrate@3": HitRate(k=3, debias_config=debias_config),
+            "debias_map@1": MAP(k=1, debias_config=debias_config),
+            "debias_map@3": MAP(k=3, debias_config=debias_config),
+            "debias_ndcg@3": NDCG(k=3, debias_config=debias_config),
+            "debias_mrr@3": MRR(k=3, debias_config=debias_config),
+            "debias_pap@3": PAP(k=3, debias_config=debias_config),
+            "debias_partauc@3": PartialAUC(k=3, debias_config=debias_config),
+        }
+        metrics = {
+            "debias_precision@3": Precision(k=3),
```

Comment:
Why these metrics called `debias_...` while they are not debiased?

Reply:
I have used the same name for consistency of dict keys, if i was to change the name it would throw an error for this test

do you have any other naming suggestions instead?

Reply:
ok, let's keep it as is